python/ray/serve/pipeline/json_serde.py

```diff
@@ -142,28 +142,22 @@ def dagnode_from_json(input_json: Any) -> Union[DAGNode, RayServeHandle, Any]:
         )
     # Deserialize DAGNode type
```

Comment:
In my next PR to introduce minimal JSON, I merged these into a simple name -> cls dispatch table. It's not in this PR for simplicity.